examples/client/client.c

```diff
@@ -792,6 +797,66 @@ static int wolfSSH_AGENT_IO_Cb(WS_AgentIoCbAction action,
 
 #endif /* WOLFSSH_AGENT */
 
+#ifdef WOLFSSH_TPM
+int wolfSSH_TPM_InitKey(TPM2_DEV *dev, const char *name, WOLFTPM2_KEYBLOB *tpmKey)
+{
+    WOLFTPM2_KEY storage;
+
+    if (wolfTPM2_Init(dev, TPM2_IoCb, NULL) != TPM_RC_SUCCESS) {
+#ifdef DEBUG_WOLFSSH
+        printf("TPM 2.0 Device initialization failed\n");
+#endif
+        return WS_FAILURE;
+    }
+
+    /* TPM 2.0 keys live under a Primary Key, acquire such key */
+    if (getPrimaryStoragekey(dev, &storage, TPM_ALG_RSA)) {
```

Comment:
We should only be using real WolfTPM public API's. Not our test key wrappers. Even if you need to move some of the code from examples/tpm_test_keys.c` into wolfSSH.

Reply:
I was wondering this. Will do.

Reply:
What to do with the IO Callbacks that are in examples/tpm_io.c(h) ?

I could just use NULL and have native Windows, Linux and TPM Simulator supports. And for baremetal to point to wolfTPM code.

Reply:
I don't mind if you use the `example/tpm_io.c` for the other IO callbacks.

Reply:
I would like to see all the code here calling wolfTPM native API's only. Bring in the `getPrimaryStoragekey` and `readKeyBlob` logic here. Eliminate the other files. I would like one copy/paste section of code as a template for this.